src/components/likedvideospage/LikedVideosPage.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useLikes } from "../../context";
+import VideoCard from "../videocard/VideoCard";
+import "./likedvideopage.css"
+
+export default function LikedVideosPage() {
+
+    const {likedVideos} = useLikes()
+
+    return <div className="grand-main">
+        <h2 className="p-y-1">Liked Videos:</h2>
```

Comment:
What is the naming convention for css class being followed ?

h2 className="p-y-1"???

Reply:
That's an utility class for padding